lib/client/client.js

```diff
@@ -238,6 +238,8 @@ Client.prototype.add = function add (name, entry, controls, callback) {
         save[k].forEach(function (v) {
           attr.addValue(v.toString())
         })
+      } else if (Buffer.isBuffer(save[k])) {
+        attr.addValue(save[k]);
```

Comment:
Please follow the linting rules. Use `npm run lint`.

Reply:
npm run lint completed

note: I have eslint installed globally, but it was not working.  So I installed it as a local dev dependency along with required peers pulled in by extends standard: eslint-plugin-import eslint-plugin-node, and eslint-plugin-promise.

Please advise as to best practices here.  Seems like these should be included as dev dependencies in the repo.

Reply:
`npm install` should install all required dev dependencies. I know we were using vanilla `standard` but that switched to support some configuration extension. Maybe we missed adding some dependencies @UziTech ?

Reply:
No, it's there.  Not sure what my problem was.  Sorry for this noise.